sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sampling_algorithm.py

```diff
@@ -58,6 +72,17 @@ def __init__(
         seed: int = None,
         logbase: Union[float, str] = None,
     ) -> None:
+        """Random Sampling Algorithm.
+
+        :param rule: The specific type of random algorithm. Possible values include: "random",
+            "sobol".
+        :type rule: str
+        :param seed: The seed for random number generation.
+        :type seed: int
+        :param logbase: A positive number or e in string format to be used as base for log
```

Comment:
nit: similar to other logbase.
```suggestion
    :param logbase: A positive number or "e" in string format to be used as the base for
        log-based random sampling.
```